tests/unit_tests/test_math.py

```diff
@@ -213,13 +214,19 @@ def test_normal_dist():
 
     assert ref_val == pytest.approx(test_val)
 
-    prn_seed = 1
-    a = 14.08
+    # make sigma = 1.0
```

Comment:
Really small thing, but perhaps we could remove the comment here and update the variable names of `a` and `b` to `mean` and `sigma` respectively.

Reply:
Good point -- I'll actually change to `mean` and `stdev`